Converted Protostar references and updated docs links (#507)
software-mansion/scarb: website/pages/docs/testing.mdx

To do so, provide a script named explicitly `test` in the current workspace `Scarb.toml`.
If such script is found, Scarb will execute it instead of running the default test runner.

> For example, in order to tell `scarb test` to use [Snforge](https://foundry-rs.github.io/starknet-foundry/index.html) for testing in

Reviewer: ```suggestion
For example, in order to tell `scarb test` to use [Starknet Forge](https://foundry-rs.github.io/starknet-foundry/index.html) for testing in
```

> For example, in order to tell `scarb test` to use [Starknet Forge](https://foundry-rs.github.io/starknet-foundry/index.html) for testing in

Reviewer 2: ```suggestion
For example, in order to tell `scarb test` to use [Starknet Forge](https://foundry-rs.github.io/starknet-foundry) for testing in
```

It is also possible to run multiple Cairo test runners, if there is a sensible use case for this.
Mind, that you will probably want to conditionally disable tests depending on the running test runner, so that tests
will not be executed multiple times.

> For example, to make `scarb test` run `scarb cairo-test` and then Snforge in a sequence, type the following:

Reviewer: ```suggestion
For example, to make `scarb test` run `scarb cairo-test` and then Starknet Forge in a sequence, type the following:
```
